package-lock.json

```diff
@@ -5,22 +5,17 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.10.1",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.1.tgz",
-      "integrity": "sha512-IGhtTmpjGbYzcEDOw7DcQtbQSXcG9ftmAXtWTu9V936vDye4xjjekktFAtgZsWpzTj/X01jocB46mTywm/4SZw==",
+      "version": "7.16.0",
```

Comment:
why we have changes for `package-lock.json` in the PR? Does any dependencies should be changed?

Reply:
@semarche-kaltura the contrib is not updated to the newest and why not update the dependencies by the way